assets/src/js/admin/reports/components/legend/index.js

```diff
@@ -1,76 +1,73 @@
+	const hideData = ( index, value ) => {
+		const inactive = inactiveItems.slice( 0 );
+		if ( chartRef.data.datasets[ 0 ].data[ index ] === value ) {
+			chartRef.data.datasets[ 0 ].data[ index ] = null;
+			inactive.push( index );
+			setInactiveItems( inactive );
+		} else {
+			chartRef.data.datasets[ 0 ].data[ index ] = value;
+			const foundIndex = inactive.indexOf( index );
+			inactive.splice( foundIndex, 1 );
+			setInactiveItems( inactive );
+		}
+		//chartRef.data.datasets[0].data[index] = chartRef.data.datasets[0].data[index] === value ? null : value
+		chartRef.update();
+	};
 
-    // Hide a given datapoint in the chart and the coresponding legend item
-    const hideData = (index, value) => {
-        let inactive = inactiveItems.slice(0)
-        console.log('hide data!')
-        if (chartRef.data.datasets[0].data[index] === value) {
-            chartRef.data.datasets[0].data[index] = null
-            inactive.push(index)
-            setInactiveItems(inactive)
-        } else {
-            chartRef.data.datasets[0].data[index] = value
-            const foundIndex = inactive.indexOf(index)
-            inactive.splice(foundIndex, 1)
-            setInactiveItems(inactive)
-        }
-        //chartRef.data.datasets[0].data[index] = chartRef.data.datasets[0].data[index] === value ? null : value
-        chartRef.update()
-    }
+	// Map data labels to build legend items
+	const items = data.labels.map( ( label, index ) => {
+		// Prepare status class for item
+		const status = inactiveItems.includes( index ) ? ' inactive' : '';
 
-    // Map data labels to build legend items
-    const items = data.labels.map((label, index) => {
+		// Use ulitity function to match item color to chart color
+		const color = getColor( index );
 
-        // Prepare status class for item
-        const status = inactiveItems.includes(index) ? ' inactive' : ''
+		// Prepare percent to display with legend item
+		const percent = Math.round( ( data.datasets[ 0 ].data[ index ] / total ) * 100 ) + '%';
 
-        // Use ulitity function to match item color to chart color
-        const color = getColor(index)
-
-        // Prepare percent to display with legend item
-        const percent = Math.round( ( data.datasets[0].data[index] / total ) * 100 ) + '%'
-
-        return (
-            <div className={'item' + status} key={index} onClick={() => hideData(index, data.datasets[0].data[index])}>
-                <div className='bar' style={{background: color}}></div>
-                <p className='label'>{label}</p>
-                <p className='data'>{data.datasets[0].data[index]}</p>
-                <p className='percent'>{percent}</p>
-            </div>
-        )
-    })
-    return (
-        <div className='givewp-legend'>
-            {items}
-        </div>
-    )
-}
+		return (
+			<div className={ 'item' + status } key={ index } onClick={ () => hideData( index, data.datasets[ 0 ].data[ index ] ) }>
```

Comment:
@henryholtgeerts   to make the code cleaner. You can create an array of classes and join them when needed.
For example:
```
className={ classes.join( ' ' ) }
```